src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,58 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Comments } from '../../types/Comments';
+import { User } from '../../types/User';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  title: string;
+  body: string;
+  user: User | null;
+  comments: Comments [];
+};
```

Comment:
```suggestion
type Props = {
  post: {
    title: string;
    body: string;
    user: User | null;
    comments: Comments [];
  }
};
```

but better in this way:
```
type Props = {
  post: Post;
}
```